README.md

```diff
@@ -1,4 +1,65 @@
-# caros-release
-CAROS release redomat declarations
+# CarOS
 
-Use [redomat](https://github.com/carosio/redomat) to build CAROS based on declarations found in this repository
+CarOS is a Linux-based OS distribution targetting carriers (thus the name). It provides modern infratructure for monitoring and logging, providing both, a run-time environment for distributed applications as well as virtualization and containerization.
+
+CarOS release is rolling and has one announcement per month with release drop (image etc.)
+
+https://github.com/carosio/caros-release/releases/latest
+
+
+## caros-release repository
+CarOS release redomat declarations
+
+Use [redomat](https://github.com/carosio/redomat) to build CarOS based on declarations found in this repository
+
+### Build 
+
+System base:
+ Ubuntu Server 15.10 only with SSH-Server 
+ 
+Install Docker:
+ (https://docs.docker.com/installation/ubuntulinux/)
+ 
+ ~~~
+ sudo apt-key adv --keyserver hkp://pgp.mit.edu:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D
+ sudo echo "deb https://apt.dockerproject.org/repo ubuntu-wily main" >> /etc/apt/sources.list.d/docker.list
+ sudo apt-get update
+ sudo apt-get install docker-engine 
+ ~~~
+ 
+Install python libs and bulidtools 
+
+~~~
+ sudo apt-get install build-essential python3-venv python-pip python-docker
+~~~
+
+Checkout redomat
+
+~~~
+ git clone https://github.com/carosio/redomat.git
+ cd redomat/
+ make
+ cd ..
+~~~
+
+Checkout caros redomat xml files
+
+~~~
+ git clone https://github.com/carosio/caros-release.git
+~~~
+
+Build vmware images
+
+~~~
+ sudo redomat/redomat.py caros-release/caros-15.10-vmware.xml
+~~~
+
+Get the file
+
+~~~
+ docker images
+ docker run --rm -ti -p <free_local_port>:80 <image ID from biggest images> /REDO/results/serve.sh
```

Comment:
biggest image? the build process outputs the image name. the last line on a successful build reads like:

```
2015-11-16 18:33:00,933 INFO: [ 2015-11-16_172039-thz-vmware 003-build-packages 66e2db5e ] end of stage actions. committing (66e2db5e): 2015-11-16_172039-thz-vmware:003-build-packages
```

That is the image name to provide as image to docker run.